src/base/Error.ts

```diff
@@ -13,6 +13,13 @@ import {Logs} from "../base/Log"
         }
     }
 
+    export class NotConnectedError extends Error {
+        constructor(message: string) {
+            super(message);
+            this.name = 'NotConnectedError'
+        }
+    }
+
```

Comment:
How about: 
export class MultichainError extends Error {
   cause: any
constructor(cause:any) {
      super("Unexpected error while calling MultichainLib function")
     this.cause = cause
   }
}

This will handle any errors that may come out of the wrapped lib; involves less string manipulation; and will not lose any information that may be available from the root cause